server/server.go

```diff
@@ -458,7 +459,12 @@ func (server HTTPServer) readAggregatorRuleForClusterID(
 	return aggregatorResponse.Report, true
 }
 
-func (server HTTPServer) fetchRuleContent(rule types.RuleOnReport) (proxy_types.RuleWithContentResponse, bool) {
+// fetchRuleContent - fetching content for particular rule
+// Return values:
+//   - Structure with rules and content
+//   - return true if fetching content was successful, including filtering
+//   - return true if fetching and parsing of content was successful, not including filtering
+func (server HTTPServer) fetchRuleContent(rule types.RuleOnReport, OSDEligible bool) (proxy_types.RuleWithContentResponse, bool, bool) {
```

Comment:
at least the last `bool` seems it should be in fact `error`

Reply:
I'm not sure about that, because `fetchRuleContent` handling errors by itself, so in that case we just need to be sure if there is some errors.